awsrte53taskgroup.go

```diff
@@ -0,0 +1,134 @@
+package ibclient
+
+type AWSRte53TaskOperations interface {
+	CreateAWSRte53TaskGroup(aws AWSRte53TaskGroup) (*AWSRte53TaskGroup, error)
+	GetAWSRte53TaskGroup(aws AWSRte53TaskGroup) (*[]AWSRte53TaskGroup, error)
+	DeleteAWSRte53TaskGroup(aws AWSRte53TaskGroup) (string, error)
+	UpdateAWSRte53TaskGroup(aws AWSRte53TaskGroup) (*AWSRte53TaskGroup, error)
+}
+
+type AWSUser struct {
+	IBBase          `json:"-"`
+	AccountID       string `json:"account_id,omitempty"`
+	AccessKeyId     string `json:"access_key_id,omitempty"`
+	LastUsed        int    `json:"last_used,omitempty"`
+	Name            string `json:"name,omitempty"`
+	NiosUserName    string `json:"nios_user_name,omitempty"`
+	SecretAccessKey string `json:"secret_access_key,omitempty"`
+	Status          string `json:"status,omitempty"`
+}
+
+// NewAWSUser creates a new AWSUser type with objectType and returnFields
+func NewAWSUser(aws AWSUser) *AWSUser {
+	res := aws
+	res.objectType = "awsuser"
+	res.returnFields = []string{"account_id", "last_used", "name", "nios_user_name", "status"}
+	return &res
+}
+
+// GetAWSUser gets the required details of the existing user
+func (objMgr *ObjectManager) GetAWSUser(aws AWSUser) (*AWSUser, error) {
+	var res []AWSUser
+	awsRte53 := NewAWSUser(aws)
+	var err error
+	err = objMgr.connector.GetObject(awsRte53, "", &res)
+	if err != nil || res == nil || len(res) == 0 {
+		return nil, err
+	}
+
+	return &res[0], err
+}
+
+type AWSRte53TaskGroup struct {
```

Comment:
```suggestion
type AWSRoute53TaskGroup struct {
```
Shall we keep it as Route it self rather than `Rte`?

Reply:
Okay, will change that as well.